packages/react-instantsearch-dom/package.json

```diff
@@ -41,7 +41,7 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.1.2",
-    "algoliasearch-helper": "0.0.0-27095c0",
```

Comment:
It's weird that we had

`"algoliasearch-helper": "0.0.0-6ac260d"` in `react-instantsearch-core` and
`"algoliasearch-helper": "0.0.0-27095c0"` in `react-instantsearch-dom`.

Anyway the changes look good to me.